code/modules/awaymissions/corpse.dm

```diff
@@ -206,9 +205,6 @@
 			var/T = vars[slot]
 			if(!isnum(T))
 				outfit.vars[slot] = T
-		if(!has_id)
-			H.equipOutfit(outfit, TRUE)
-		else
 			H.equipOutfit(outfit)

```

Comment:
YOu need to move this back....or else it's going to equip the outfit multiple times for all your slots.

Reply:
Accured tabbing!